parsec/core/gui/greet_user_widget.py

```diff
@@ -240,7 +240,8 @@ def __init__(self, jobs_ctx, greeter):
         self.line_edit_device.validity_changed.connect(self.check_infos)
         self.line_edit_device.set_validator(validators.DeviceNameValidator())
 
-        # self.combo_profile.addItem(_("TEXT_USER_PROFILE_OUTSIDER"), UserProfile.OUTSIDER)
+        if outsider_enabled:
+            self.combo_profile.addItem(_("TEXT_USER_PROFILE_OUTSIDER"), UserProfile.OUTSIDER)
```

Comment:
It would be better to have this greyed out with a popup help tip showing up when passing the mouse on it.

Of course the current solution is good enough for now (and this PR big enough !), so can you open an issue about this once this PR is merged ?

Reply:
Should look like something like this:
```python
model = QStandardItemModel()
self.combo_profile.setModel(model)
self.combo_profile.addItem("1")
self.combo_profile.addItem("2")
self.combo_profile.addItem("3")
item = model.item(2)
item.setEnabled(False)
item.setToolTip("Disabled because backend does not allow it in its current configuration")
```